### PR TITLE
Remove targetpath from gir

### PR DESCRIPTION
**Pull request by EPashkin**
WIP
Need fix gir to remove `mkdir -p $(@D)/src`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,12 +1,12 @@
 GIR = gir/target/release/gir
 CONFIGS = $(wildcard conf/gir-*.toml)
-LIBS = $(CONFIGS:conf/gir-%.toml=%-sys/src/lib.rs)
+LIBS = $(CONFIGS:conf/gir-%.toml=%-sys/Cargo.toml)
 
 libs : $(LIBS)
 
-%-sys/src/lib.rs : conf/gir-%.toml $(GIR)
+%-sys/Cargo.toml : conf/gir-%.toml $(GIR)
```
Comment (gkoz) [inline Makefile]: This replacement has a potential to produce surprising results: failure to regenerate if you edit both `foo-sys/Cargo.toml` and `conf/gir-foo.toml`.

Comment (gkoz) [inline Makefile]: We can use the `$*` variable to get at the directory name instead:
```make
%-sys/src/lib.rs : conf/gir-%.toml $(GIR)
    mkdir -p $(@D)
    $(GIR) -c $< -o $*-sys -m sys
```

**Comment by EPashkin** [comment]
Undo change to build cargo.toml
Make absolute path
When gkoz/gir#93 merged, I update gir and remove `mkdir`

**Comment by gkoz** [comment]
Out of curiosity, what happens without `abspath`?

**Comment by EPashkin** [comment]
Old reason was remove '\..' from path.
Now just not sure what do `create_dir_all` when path relative, and not exists.

**Comment by EPashkin** [comment]
And prefer full path as output

**Comment by EPashkin** [comment]
Ready.

**Comment by gkoz** [comment]
Thanks!

**Comment by EPashkin** [comment]
You plan add build test?

**Comment by gkoz** [comment]
Not yet. I consider this a prototype, further extension should wait until @GuillaumeGomez has a chance to look at all this. Currently I'm working on making the "normal" crates build with these.

**Comment by EPashkin** [comment]
IMHO better leave sys in normal crate (as current)

**Comment by EPashkin** [comment]
On other side, sys modules will be updated rarely, so may be good that it all in one place.

**Pull request merged**